docs/blog/_posts/2020-07-27-26th-dotty-milestone-release.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.26.0-RC1 - unified extension methods and more
+author: Anatolii Kmetiuk
+authorImg: /images/anatolii.png
+date: 2020-07-27
+---
+
+Hello! We are excited to announce 0.25.0-RC2 of Dotty. In this version, we have improved the extension methods – their syntax is now more uniform. We have also implemented local selectable instances and have done a bunch of improvements to the compiler and the language API. Otherwise, we are focusing our efforts on reducing the issue count on the issue tracker, boosting performance and improving the stability of the compiler in other ways.
+
+You can try out this version right now, from the comfort of your SBT, by visiting the [home page](https://dotty.epfl.ch/) and scrolling down to the "Create a Dotty Project" section.
+
+Alternatively, you can try this version of Scala online via [Scastie](https://scastie.scala-lang.org/). Once you're there, click "Build Settings" and set "Target" to "Dotty".
+
+Enjoy the ride🚀!
+
+<!--more-->
+# Unified extension methods
+In this release, we have made extension method syntax uniform. Previously, we had three separate syntaxes for single extension methods, collective extension methods and given instances with extension methods. Now, these three cases have been unified into one.  The new syntax looks like follows:
+
+```scala
+extension (x: String)
+  def < (y: String): Boolean = ...
+```
+
+Collective extensions look like follows:
+
+```scala
+extension (ss: Seq[String]):
+
+  def longestStrings: Seq[String] =
+    val maxLength = ss.map(_.length).max
+    ss.filter(_.length == maxLength)
+
+  def longestString: String = longestStrings.head
+```
+
+You can read more about the new syntax in the [documentation](https://dotty.epfl.ch/docs/reference/contextual/extension-methods.html). For the discussion, see [PR](https://github.com/lampepfl/dotty/pull/9255).
+
+# Local Selectable Instances
+Local and anonymous classes that extend `Selectable` get more refined types than other classes. For example:
+
+```scala
+class Vehicle extends reflect.Selectable {
```

Comment:
```suggestion
trait Vehicle extends reflect.Selectable {
```
? Or `abstract class`?

Reply:
Good catch. Also updated it in the docs.